tests/phpunit/tests/rest-api/rest-attachments-controller.php

```diff
@@ -215,10 +215,8 @@ public function test_registered_query_params() {
 			'video',
 			'image',
 			'audio',
+			'text',
 		);
-		if ( ! is_multisite() ) {
```

Comment:
@spacedmonkey is there some reason text attachments were specifically disallowed in multisite?

Reply:
This one confused me here. The amount of file that can be uploaded to multisite, seems very limited and for no good reason I can understand.

Reply:
To answer my own question, I doin't think so. This test was added in https://github.com/WordPress/wordpress-develop/commit/ede099a7047a150a37d43a380e661b99387bce41 and I think the code is just meant to verify the current state of affairs, where text was not added in multisite.